src/renderer/widgets/AddProxyModal/ui/AddProxy.tsx

```diff
@@ -43,7 +43,12 @@ export const AddProxy = () => {
       {addProxyUtils.isConfirmStep(step) && (
         <Confirmation
           secondaryActionButton={
-            <AddToBasketButton wallet={initiatorWallet} onTxSaved={() => addProxyModel.events.txSaved()} />
+            initiatorWallet &&
+            !basketUtils.isBasketAvailable(initiatorWallet) && (
```

Comment:
I suppose it's `initiatorWallet  && basketUtils.isBasketAvailable(initiatorWallet)` 🤔